src/js/util.js

```diff
@@ -965,6 +964,18 @@
             } else {
                 el.parentNode.removeChild(el);
             }
+        },
+
+        // https://github.com/Voog/wysihtml/issues/231
+        // fixes IE 11 bug with removeAllRanges
+        replaceRange: function (sel, replacement) {
+            if (sel.rangeCount > 0 &&
+                sel.getRangeAt(0).getClientRects.length < 0) {
```

Comment:
As per the link you mentioned here, I believe this should be `sel.getRangeAt(0).getClientRects.length > 0` no?